src/components/LayerControls.tsx

```diff
@@ -0,0 +1,32 @@
+import { CalciteSlider, CalciteSwitch } from '@esri/calcite-components-react';
+import { CalciteSliderCustomEvent, CalciteSwitchCustomEvent } from "@esri/calcite-components";
+
+interface LayerControlsProps {
+    layerVisibility: boolean | undefined;
+    handleVisibilityToggle: (event: CalciteSwitchCustomEvent<void>) => void | undefined;
+    layerOpacity: number;
+    handleOpacityChange: (e: CalciteSliderCustomEvent<void>) => void;
+}
+
+const LayerControls: React.FC<LayerControlsProps> = ({ layerVisibility, handleVisibilityToggle, layerOpacity, handleOpacityChange }) => (
```

Comment:
A controls component that can takes in some handlers that affect whichever layer we want to toggle visibility/opacity for